### DISCUSSION TITLE
Airlock Browser Feature Requests

### DISCUSSION THREAD

**Discussion by DVaughan** [OutcoderSoftware/AirlockBrowser#8] [Ideas]
### Welcome to the Airlock Browser Feature Requests discussion!

This is the dedicated space where you can share your ideas for new features or enhancements for Airlock Browser. We encourage you to be as detailed as possible so others can understand and provide valuable feedback.

#### Please Note:

If you wish to report an issue or a bug, kindly use our [Issues page](https://github.com/OutcoderSoftware/AirlockBrowser/issues).

By submitting an idea here, you'll give others the opportunity to comment and discuss, helping us to prioritize our work based on community input.

We look forward to hearing your innovative ideas and working together to improve our product!

**Comment by KatkaVaughan** [comment]
Adding Client-Side CSS to Pages - When adding CSS in Application Profile could we have an option like the one for JavaScript "Append to body" but call it "Append to head" so the CSS  is not invoked but added directly to the head of the html page. This may solve the issue when users see the transition from the original style to the new one, 

Maybe this resource may help: https://stackoverflow.com/questions/67754291/is-there-a-way-to-have-an-webview-on-android-where-i-can-inject-css-before-the-p
